components/Report/data_fetcher_and_renderer.rst

```diff
@@ -0,0 +1,35 @@
+DefaultDataFetchers
+===================
+
+.. hint::
+    This default data fetchers are implemented in `SyliusCoreBundle`_.
+    They extend `TimePeriod`_.
+
+.. _TimePeriod: http://api.sylius.org/Sylius/Bundle/ReportBundle/DataFetcher/TimePeriod.html
+.. _SyliusCoreBundle: http://api.sylius.org/Sylius/Bundle/CoreBundle/DataFetcher.html
+
++-------------------+-------------------+-----------------------------------------------+
+| Related constant  | Data Fetcher      | Description                                   |
++===================+===================+===============================================+
+| USER_REGISTRATION | user_registration | Fetch all registered users in specific period |
++-------------------+-------------------+-----------------------------------------------+
+| SALES_TOTAL       | sales_total       | Fetch all sales in specific period            |
++-------------------+-------------------+-----------------------------------------------+
+| NUMBER_OF_ORDERS  | number_of_orders  | Fetch all number of orders in specific period |
++-------------------+-------------------+-----------------------------------------------+
+
+DefaultRenderers
+================
+
+.. hint::
+    This default renderers are implemented in `SyliusReportBundle`_.
+
+.. _SyliusReportBundle: http://api.sylius.org/Sylius/Bundle/ReportBundle/Renderer.html
+
++------------------+----------+------------------------+
+| Related constant | Renderer | Description            |
++==================+==========+========================+
+| TABLE            | table    | Default table renderer |
++------------------+----------+------------------------+
+| CHART            | chart    | Default chart renderer |
++------------------+----------+------------------------+
```

Comment:
I think we should remove these, at least from documentation for now and avoid suggesting to use them. Because one may apply different report types etc. It is ok to keep current examples Table and SalesTotal - just keep the constants inside implementations. Remove this whole file. ;)